src/components/dropdown/dropdown.js

```diff
@@ -648,6 +652,19 @@ class Dropdown extends React.Component {
 
     return results;
   }
+  /**
+   * positions the portal listBlock in relation to the input.
+   *
+   * @method calculatePosition
+   * @return {void}
+   */
+  calculatePosition() {
```

Comment:
change this to

```
calculatePosition = () => {
```

this uses class properties, and automatically binds the instance to the method.